pkg/types/baremetal/defaults/platform.go

```diff
@@ -0,0 +1,67 @@
+package defaults
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/openshift/installer/pkg/types"
+	"github.com/openshift/installer/pkg/types/baremetal"
+)
+
+// Defaults for the baremetal platform.
+const (
+	LibvirtURI         = "qemu:///system"
+	IronicURI          = "http://localhost:6385/v1"
+	ExternalBridge     = "baremetal"
+	ProvisioningBridge = "provisioning"
+	HardwareProfile    = "default"
```

Comment:
Ah so here we are setting the hardwareProfile. Why not use DefaultProfileName ?

Reply:
https://github.com/openshift/installer/pull/1873#discussion_r304056325

Reply:
we could, note that we still need other code that translates "default" into DefaultProfileName though.  That's what the other code handles